app/scripts/metamask-controller.js

```diff
@@ -917,9 +923,13 @@ export default class MetamaskController extends EventEmitter {
         ),
       getCurrentAccountEIP1559Compatibility:
         this.getCurrentAccountEIP1559Compatibility.bind(this),
-      getNetworkState: () => this.networkController.store.getState().network,
+      getNetworkId: () => this.networkController.store.getState().networkId,
+      getNetworkStatus: () =>
+        this.networkController.store.getState().networkStatus,
+      // TODO: This option should probably listen for networkDidChange
```

Comment:
This was weird to me, but not necessary to fix in this PR. I'll make a new issue.

Reply:
Comment removed [here](https://github.com/MetaMask/metamask-extension/pull/17556/commits/97727529dc0462b3b01bdd43f6b1d4a5d28dcb78) and issue added [here](https://github.com/MetaMask/metamask-extension/issues/18291).